api_v3/lib/types/partner/KalturaPartner.php

```diff
@@ -444,7 +445,24 @@ public function toPartner()
 		
 		return $partner;
 	}
-	
+
+	/* (non-PHPdoc)
+	 * @see KalturaObject::toObject()
+	 */
+	public function toObject($dbPartner = null, $skip = array())
+	{
+		if(!$dbPartner)
+		{
+			$dbPartner = new Partner();
+		}
+
+		if ($this->describeYourself && (strlen ($this->describeYourself) > self::MAX_DESCRIBE_YOURSLEF_LEN))
+		{
+			$this->describeYourself = kString::alignUtf8String($this->describeYourself, 64);
```

Comment:
1. Use const here (instead of 64)
2. Apply this fix to all the relevant fields that can have an issue here.